packages/botonic-plugin-dialogflow/src/types.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+//Dialogflow types
+export class Credentials {
+  private_key_id: string
+  private_key: string
+  client_email: string
+  project_id: string
+  constructor(
+    private_key_id: string,
+    client_email: string,
+    private_key: string,
+    project_id: string
+  ) {
+    this.private_key_id = private_key_id
+    this.client_email = client_email
+    this.private_key = private_key
+    this.project_id = project_id
+  }
+}
+
+export class Request {
```

Comment:
you could declare @botonic/core as devdependency to import this type (which is actually the interface BotRequest)